src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="BuildPropertyPage"
+      Description="General"
+      DisplayName="Build"
+      PageTemplate="generic"
+      xmlns="http://schemas.microsoft.com/build/2009/properties" >
+
+  <Rule.Categories>
+    <Category Name="General"
+              Description="General"
+              DisplayName="General" />
+
+    <Category Name="ErrorsAndWarnings"
+              Description="Errors and warnings"
+              DisplayName="Errors and warnings" />
+
+    <Category Name="Output"
+              Description="Output"
+              DisplayName="Output" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <StringProperty Name="DefineConstants"
+                  DisplayName="Conditional compilation symbols"
+                  Category="General" />
+
+  <EnumProperty Name="PlatformTarget"
```

Comment:
I noticed in the screenshots that the Platform and Nullable properties didn't show a value. I suspect this is because the .props/.targets don't provide an explicit default value, and so we can't get the default from MSBuild evaluation.

Can you check the documentation to see if there is a way to specify the default value here in the .xaml file?

Reply:
I tried with the EnumProperty's  `Default` property and the EnumValue's `IsDefault` property, but had no luck.